v23.1/monitor-and-debug-changefeeds.md

```diff
@@ -59,6 +59,17 @@ By default, changefeeds will retry errors with [some exceptions](#changefeed-ret
 - `changefeed.error_retries`: The total number of retryable errors encountered by all changefeeds.
 - `changefeed.failures`: The total number of changefeed jobs that have failed.
 
+#### Protected timestamp and garbage collection monitoring
+
+Protected timestamps will protect changefeed data from garbage collection in particular scenarios, but if a changefeed lags too far behind, the protected changes could cause data storage issues. See [Garbage collection and changefeeds](changefeed-messages.html#garbage-collection-and-changefeeds) for detail on when changefeed data is protected from garbage collection.
```

Comment:
suggest first mention of protected timestamps be a link to the pts stuff in architecture docs

I think "could cause data storage issues" would be more impactful if more concrete, something like "the protected changes will cause an accumulation of [MVCC values](https://www.cockroachlabs.com/docs/v23.1/architecture/storage-layer.html#mvcc) which in addition to increasing disk usage will negatively impact writes to those values, including DELETEs" ... or similar?  you would probably want an engineer to vet that text so understand if you need to punt for now given release deadline